content/landing_pages/divingDeeperPlatforms.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Multiple Platforms
+image: 
+description: Learn all using Babylon.js on multiple platforms.
+keywords: diving deeper, cross-platform, platform
+further-reading:
+video-overview:
+video-content:
+---
+
+[![Web app](/img/flatIcons/Web-256.png!128)](#web-app) &nbsp &nbsp [![Mobile app](/img/flatIcons/Ipad-256.png!128)](#mobile-app) &nbsp &nbsp [![Desktop app](/img/flatIcons/Computer-Desktop-256.png!128)](#desktop-app)
+
+(Royalty-free icons from [Flaticons.net](https://flaticons.net/).)

```

Comment:
I'm not sure you really need this. The docs automatically generate a TOC, so this is duplicate info? Also, it's hard to understand what is happening with the icons.

<img width="243" alt="image" src="https://user-images.githubusercontent.com/19158740/163218287-808dd474-aa2a-4419-aab5-5c90ade265ab.png">

Reply:
Funky. Locally, the page renders to have the icons look like this:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/33846034/163237118-a5491312-58a0-42ae-8953-cae0aae93718.png">

Not sure why Vercel shows it different from this, but the purpose of the icons is just to visually break up what would otherwise be an immediate wall of text as soon as the page loads. Not a make-or-break issue, but when they render correctly I do think they make the page more friendly. Spoke to Jason about it, and we think the best plan is to leave them in for now and, if they also render wrong in the production site, ask Raanan to help us figure that out when he gets back.

Reply:
> the purpose of the icons is just to visually break up what would otherwise be an immediate wall of text as soon as the page loads

Maybe don't make them links then. It might be just me, but I'm not sure what the icons mean by looking at them. It's also weird that the first piece of non-TOC text is "Royalty-free icons ...".

Reply:
@RaananW to take a look

Reply:
can you merge from master and publish a new version? I will be able to check the preview and the way the page was built.

Having said that, I agree with Gary - it feels weird that those icons are there, especially since it's the only page that has these kinds of links to the content.

Reply:
I talked to Raanan, we're going to remove the icons for now and revisit later.

@syntheticmagus - Can you please remove these?